packages/create-discord-bot/src/create-discord-bot.ts

```diff
@@ -62,9 +62,7 @@ export async function createDiscordBot({ typescript, javascript, directory }: Op
 		process.exit(1);
 	}
 
-	if (!directoryStats.isDirectory()) {
-		await mkdir(root, { recursive: true });
-	}
+	await mkdir(root, { recursive: true });
```

Comment:
Still redundant, this line doesn't do anything except for throwing an error if a file (not a directory) of this name already exists, in which case it would be nice to have an error message.